src/realm/util/file.cpp

```diff
@@ -1045,10 +1045,10 @@ void File::barrier()
     // If F_BARRIERSYNC is not suppported (this is known on exFAT) we don't fallback to sync().
```

Comment:
can you change this comment back as well?

Reply:
ah, sorry.. yes, you are right.